src/DynamoCore/Visualization/DefaultRenderPackageFactory.cs

```diff
@@ -156,96 +217,166 @@ public void Clear()
             DisplayLabels = false;
         }
 
+        /// <summary>
+        /// A tag used to store information about the render package.
+        /// </summary>
         public string Description { get; set; }
 
+        /// <summary>
+        /// A flag indicating whether the render package is selected.
+        /// </summary>
         public bool IsSelected { get; set; }
 
+        /// <summary>
+        /// A flag indicating whether the render package has data.
+        /// </summary>
         public bool HasRenderingData
         {
             get { return pointVertices.Any() || meshVertices.Any() || lineVertices.Any(); }
         }
 
+        /// <summary>
+        /// A flag indicating whether the render package is displaying labels
+        /// </summary>
         public bool DisplayLabels { get; set; }
 
+        /// <summary>
+        /// A flag indicating whether the render package requires 
+        /// per vertex coloration.
+        /// </summary>
         public bool RequiresPerVertexColoration { get; set; }
 
+        /// <summary>
+        /// The number of point vertices in the package.
```

Comment:
Add Returns here. It should be Returns the number of....
correct all the below ones.